### ISSUE TITLE
What the heck is going on here?

### ISSUE THREAD

**Issue by Goldstndrd** [oscardelben/RailsOneClick#11]
linking shared-object digest.bundle
installing digest libraries
installing default digest libraries
compiling bubblebabble.c
linking shared-object digest/bubblebabble.bundle
installing default bubblebabble libraries
compiling md5init.c
md5init.c:16: warning: 'MD5_Init' is deprecated (declared at /usr/include/openssl/md5.h:113)
md5init.c:17: warning: 'MD5_Update' is deprecated (declared at /usr/include/openssl/md5.h:114)
compiling md5ossl.c
md5ossl.c: In function 'MD5_Finish':
md5ossl.c:8: warning: 'MD5_Final' is deprecated (declared at /usr/include/openssl/md5.h:115)
linking shared-object digest/md5.bundle
ld: in '/usr/local/lib/libz.1.dylib', file was built for unsupported file format ( 0xce 0xfa 0xed 0xfe 0x 7 0x 0 0x 0 0x 0 0x 3 0x 0 0x 0 0x 0 0x 6 0x 0 0x 0 0x 0 ) which is not the architecture being linked (x86_64): /usr/local/lib/libz.1.dylib for architecture x86_64
collect2: ld returned 1 exit status
make[2]: **\* [../../../.ext/x86_64-darwin12.4.0/digest/md5.bundle] Error 1
make[1]: **\* [ext/digest/md5/all] Error 2
make: **\* [build-ext] Error 2

**Comment by oscardelben** [comment]
what version of mac os x are you using?

**Comment by Goldstndrd** [comment]
10.8.4 Mountain Lion

ERROR:  Loading command: update (LoadError)
cannot load such file -- zlib
ERROR:  While executing gem ... (NoMethodError)
    undefined method `invoke_with_build_args' for nil:NilClass

**Comment by Goldstndrd** [comment]
it seems impossible to get rails, everytime i go to get it it says zlib
error. I even loaded zlib.

**Comment by oscardelben** [comment]
I will try to figure out what's going on.

**Comment by willkessler** [comment]
Something similar just now, OSX 10.7.5 macbook pro.

tcltklib.c: In function 'lib_conv_listelement':
tcltklib.c:9958: warning: implicit conversion shortens 64-bit value into a 32-bit value
tcltklib.c:9965: warning: implicit conversion shortens 64-bit value into a 32-bit value
tcltklib.c:9968: warning: implicit conversion shortens 64-bit value into a 32-bit value
tcltklib.c: In function 'tcltklib_compile_info':
tcltklib.c:10032: warning: implicit conversion shortens 64-bit value into a 32-bit value
linking shared-object tcltklib.bundle
ld: in /usr/local/lib/libiconv.2.dylib, missing required architecture x86_64 in file for architecture x86_64
collect2: ld returned 1 exit status
make[2]: **\* [../../.ext/x86_64-darwin11.4.2/tcltklib.bundle] Error 1
make[1]: **\* [ext/tk/all] Error 2
make: **\* [build-ext] Error 2

Any suggestions?

**Comment by oscardelben** [comment]
Hi, not at this time. My guess is that you need to install Xcode's developer tools.

**Comment by willkessler** [comment]
Yeah, that didn't help, I have all of Xcode incl. cmdline tools intalled. (Am able to run gcc, etc, build Xcode objective c projects, etc).

This is a clue though:

> sudo gem install rails
> Building native extensions.  This could take a while...
> ERROR:  Error installing rails:
>         activesupport requires Ruby version >= 1.9.3.

So i guess I need to figure out how to upgrade my installed Ruby, which is, for this verson of macos, 

ruby 1.8.7 (2012-02-08 patchlevel 358) [universal-darwin11.0]

**Comment by oscardelben** [comment]
You need to use the app to open a terminal window in order to use Rails, otherwise the sandbox wont work and it will use the default Ruby.

**Comment by willkessler** [comment]
That's ok, I gave up on macos and got everything working on ubuntu on AWS
without any issues.

On Mon, Aug 26, 2013 at 12:18 PM, Oscar Del Ben notifications@github.comwrote:

> You need to use the app to open a terminal window in order to use Rails,
> otherwise the sandbox wont work and it will use the default Ruby.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/oscardelben/RailsOneClick/issues/11#issuecomment-23287690
> .

## 

Will Kessler
415 203-8627 cell
415 921-2644 home
